tool/grind/standalone.dart

```diff
@@ -21,20 +22,30 @@ snapshot() {
   Dart.run('bin/sass.dart', vmArgs: ['--snapshot=build/sass.dart.snapshot']);
 }
 
-@Task('Build Dart application snapshot.')
-appSnapshot() {
+@Task('Build a dev-mode Dart application snapshot.')
+appSnapshot() => _appSnapshot(release: false);
+
+@Task('Build a release-mode Dart application snapshot.')
+releaseAppSnapshot() => _appSnapshot(release: true);
+
+/// Compiles Sass to an application snapshot.
+///
+/// If [release] is `true`, this compiles in checked mode. Otherwise, it
+/// compiles in unchecked mode.
+void _appSnapshot({@required bool release}) {
+  var args = [
+    '--snapshot=build/sass.dart.app.snapshot',
+    '--snapshot-kind=app-jit'
+  ];
+  if (!release) args..add('--checked');
```

Comment:
TODO to remove `--checked` once on Dart 2?

Reply:
Done.